src/Features/LanguageServer/Protocol/Handler/Symbols/DocumentSymbolsHandler.cs

```diff
@@ -49,54 +46,64 @@ public override async Task<object[]> HandleRequestAsync(DocumentSymbolParams req
                 return Array.Empty<SymbolInformation>();
             }
 
-            var symbols = ArrayBuilder<object>.GetInstance();
-
-            var navBarService = document.Project.LanguageServices.GetRequiredService<Editor.INavigationBarItemService>();
```

Comment:
_Does not have to be done in this PR_, but can the editor version now disappear, or is it still around for IVT stuff with F#/TypeScript?

Reply:
@CyrusNajmabadi?

Reply:
yeah, we still need it for them.